src/transactions/transferFeedUtils.ts

```diff
@@ -123,6 +123,7 @@ export function useTransferFeedDetails(transfer: FeedTokenTransfer) {
   const tokenInfo = useTokenInfoByAddress(transfer.amount.tokenAddress)
   const coinbasePaySenders = useSelector(coinbasePaySendersSelector)
   const fcTransferDisplayInfo = useFiatConnectTransferDisplayInfo(transfer)
+  const isJumpstart = isJumpstartTransaction(transfer)
```

Comment:
```suggestion
  const isJumpstartTransaction = isJumpstartTransaction(transfer)
```
super nit 😅

Reply:
I can't use that name, because the function is called like that. 😅 
I hope leaving `isJumpstart` is fine.